Robert

robscafe433 opened the pull request.

robscafe433 committed:
change howtoplaymodal.css path name

Per linting recomendation changed path name to './howtoplaymodal.css'

robscafe433 committed:
from howtoplaymodal.js move show,handleClose, and handleShow to App.js

 moved show, ha ndleClose, and handleShow to App.js (the smart component) so as to be accesbile to other components via context.

robscafe433 committed:
move state-show and function-handle close/show to parent component App.js

moved state-show and functions handle close and handle show to parent component so asto be able to access these via context to component navbar.js

robscafe433 committed:
Merge branch 'main' of https://github.com/VelazquezLuis/reactdle into robert

robscafe433 committed:
Add Sound Icons

removed sound switch and added sound icons in navbar. On and off speaker icons.

robscafe433 committed:
Change icons when clicked

sound icons in navbar now change while clicked to represent sound on and off

robscafe433 committed:
working on ways to get music automatically play on load - maybe hood useEffect.

robscafe433 committed:
Pull from main

after Felipe s pr was merged i pulled from main.

VelazquezLuis reviewed (commented):
Hi Robert are you able to fix lint issues?

robscafe433 committed:
removed line of code ESLINT_NO_DEV_ERRORS=false. This allows me to work on code before applying linting.

robscafe433 committed:
Merge branch 'main' of https://github.com/VelazquezLuis/reactdle into robert

robscafe433 committed:
add handlePlaySoundOn function to useEffect

Added handlePlaySound function to useEffect with nothing in the dependancy array, this will cause function to only run once. To only play music once modal is closed.
